mico-admin/src/app/data-view/mico-data-string/mico-data-string.component.html

```diff
@@ -1,3 +1,12 @@
-<mat-list-item class="mat-list-item">
-    <b>{{config.title}}:</b> &nbsp; {{dataValue}}
-</mat-list-item>
+<!-- Reconsider the boundary -->
+<div *ngIf="dataValue?.length > 100; else elseBlock">
+    <mat-list-item style="height: auto; min-height: 48px">
+        <b>{{config.title}}:</b> &nbsp; {{dataValue}}
+    </mat-list-item>
+</div>
+
+<ng-template #elseBlock>
+    <mat-list-item style="height: auto; min-height: 48px; display: inline-flex; vertical-align: middle">
```

Comment:
else block seems unecessary. use ngStyle to dynamically change stayle depending on data and check if added styles influence single line text.